### PR TITLE
Added the argmin and argmax function expr.py file and series.py file

### PR DESCRIPTION
**Pull request by MUKESHRAJMAHENDRAN**
## What type of PR is this? (check all applicable)

- [ ] 💾 Refactor
- [x] ✨ Feature
- [] 🐛 Bug Fix
- [x] 🔧 Optimization
- [x] 📝 Documentation
- [ ] ✅ Test
- [ ] 🐳 Other

## Related issues

- Related issue #\<issue number\>
- Closes #\<issue number\>

## Checklist

- [x] Code follows style guide (ruff)
- [ ] Tests added
- [ ] Documented the changes

### CODE REVIEW

File: docs/api-reference/expr.md [hunk 1]
```diff
@@ -19,6 +19,8 @@
         - diff
         - drop_nulls
         - ewm_mean
+        - arg_min
+        - arg_max
```
Comment (MarcoGorelli) [inline docs/api-reference/expr.md:23]: these should be sorted alphabetically

**Review by MarcoGorelli — COMMENTED** [review]
thanks, this is a good start - could you add a test please?

**Comment by MarcoGorelli** [comment]
hey - i'd suggest reading through https://narwhals-dev.github.io/narwhals/how_it_works/, i get the impression that this PR is involving a lot of guesswork

**Comment by MUKESHRAJMAHENDRAN** [comment]
Thanks, yup Mostly going through the flow thanks for the suggestion i will go through it and try to understand.

**Comment by MUKESHRAJMAHENDRAN** [comment]
Hi, I am going through the code lost myself, please guide me what is the mistake i am doing now, how should i correct it where is the issue. So i can understand the codebase better.

**Comment by MarcoGorelli** [comment]
i'd suggest looking at similar functions and understanding how they're implemented

**Comment by MarcoGorelli** [comment]
thanks for your pr

closing for now as I think you still need to understand the codebase a bit better - feel free to submit a PR once you can get the tests to pass

**Pull request closed without merging**